packages/eslint-plugin/src/rules/lines-around-comment.ts

```diff
@@ -448,6 +394,8 @@ export default util.createRule<Options, MessageIds>({
                 before: options.beforeBlockComment,
               });
             }
+          } else if (token.type === AST_TOKEN_TYPES.Shebang) {
+            // do we want to do anything here?
           }
```

Comment:
ESlint core checks for empty line here:
https://github.com/eslint/eslint/blob/main/lib/rules/lines-around-comment.js#L456

But I think we don't need to do anything here – shebang comments should trigger parser errors if used anywhere beside the first line and first line is covered by the base rule.

Reply:
Agreed, yeah, this looks good to me to leave out.